specification/dartLangSpec.tex

```diff
@@ -5991,19 +5969,23 @@ \subsection{Object Identity}
 \subsection{Constants}
 \LMLabel{constants}
 
-\commentary{
-All usages of the word 'constant' in Dart are associated with compile time.
+\commentary{%
+All usages of 'constant' in Dart are associated with compile time.
 A potentially constant expression is an expression that will generally yield
 a constant value when the value of certain parameters is given.
```

Comment:
Should this be "the values of certain parameters are given"?

Reply:
Done.